### ISSUE TITLE
General Project Discussion

### ISSUE THREAD

**Issue by Pipazoul** [Antharia/awesome-lowtech#2]

**Comment by perguth** [comment]
Nice, maybe start by outlining how to contribute, I would be interested! <3

**Comment by perguth** [comment]
I would do bump significant changes using GH releases. So people could subscribe to the low-noise "Release only" watch mode.

**Comment by Pipazoul** [comment]
Can you rephrase your message ?, Unfortunately I understood it partially

**Comment by hyphenrf** [comment]
Not sure where this fits:
https://gemini.circumlunar.space/

but it's a very simple spec, easy to implement low-power clients and servers for it

**Comment by Pipazoul** [comment]
> Not sure where this fits:
> https://gemini.circumlunar.space/
> 
> but it's a very simple spec, easy to implement low-power clients and servers for it

Does it pass the [required criterias]( https://github.com/Antharia/awesome-lowtech/blob/master/.github/CONTRIBUTING.md) ?
If yes there could be a "Protocoll"section (@Antharia )

**Comment by hyphenrf** [comment]
> Is the final weight or data stream lower than actual more "mainstream" services

I'm not sure how to verify this

> Can a relative install it, use it ?

many clients ([MasterQ32/Kristall](https://github.com/MasterQ32/Kristall)) work just like a browser would. there were even fruitful[1] efforts to create search engines for easier discoverability. If the software is packaged, installing it is as easy as a click or a command.

> Is it autonomous and not dependant to other platforms ?

depends on how dependent is too dependent; the protocol relies on existing technology like IP/DNS/etc.. network stack and TLS is a requirement AFAIK.

> Is it robust and durable ?

It's not P2P. But it'll be around as long as the internet is, and there are clients and servers.

The answer to the rest is yes.

----
[1] gemini://gus.guru/

**Comment by 0b11stan** [comment]
Thank you for the discovery!
To me it looks like the final weight and datastream of a javascriptless protocol is definitly lower than the "mainstream" web.

**Comment by hyphenrf** [comment]
Indeed. I guess a client has the complete freedom to implement a scripting language that dynamically alters a served page though if they wish.

The emphasis, I think, is in the fact that this protocol will work fine on low bandwidth and very low-powered servers, and is designed in a way that tries to push back feature-creep. which makes it way more sustainable than modern web and able to fill that communication niche without trying to extract simplicity from a big and complicated standard.
